packages/@uppy/transloadit/src/index.js

```diff
@@ -743,9 +743,8 @@ module.exports = class Transloadit extends Plugin {
 
   _onTusError (err) {
     if (err && /^tus: /.test(err.message)) {
-      const url = err.originalRequest && err.originalRequest.responseURL
-        ? err.originalRequest.responseURL
-        : null
+      const xhr = err.originalRequest ? err.originalRequest.getUnderlyingObject() : null
+      const url = xhr && xhr.responseURL ? xhr.responseURL : null
```

Comment:
tus-js-client v2 introduces a HTTP abstraction layer, so the code to access the URL is a bit different now.